components/MatchByColor/omnivoreUtils.js

```diff
@@ -1,6 +1,6 @@
 export const omnivoreUtils =  {
```

Comment:
This file is going to get deleted so do not use it, you should be using the characters js file instead.